livingatlas/pipelines/src/main/java/au/org/ala/pipelines/parser/UncertaintyParser.java

```diff
@@ -4,13 +4,16 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import lombok.extern.slf4j.Slf4j;
+import org.elasticsearch.common.Strings;
+import org.gbif.common.parsers.core.ParseResult;
+import org.gbif.common.parsers.geospatial.MeterRangeParser;
 
 /**
  * This is a port of
  * https://github.com/AtlasOfLivingAustralia/biocache-store/blob/master/src/main/scala/au/org/ala/biocache/parser/DistanceRangeParser.scala
  */
 @Slf4j
-public class DistanceRangeParser {
+public class UncertaintyParser extends MeterRangeParser {
```

Comment:
Im not sure i understand the name change. I dont think theres anything inherent about the values that are specific to coordinateUncertaintyInMeters. I suggest we change back to DistanceRangeParser as this also helps with traceability with biocache-store code..

Reply:
There are some specific ways, like range 1km-10km, Uncertainty always uses the second number. Since GBIF parser already handle UOM conversion, ours is more like Uncertainty specific implementation. Having said that, I am OK with both names.